tests/test_integration/stencil_definitions.py

```diff
@@ -205,17 +205,6 @@ def horizontal_diffusion(in_field: Field3D, out_field: Field3D, coeff: Field3D):
         )
 
 
-@register
```

Comment:
This test is very useful, but we should fix the k size or better for the GridTools backends generate an extra splitter for large offsets, because apparently GridTools is too demanding: from this code snippet I would expect a minimum size of 16 is enough, but GridTools needs 20, see slack comment from Anton.

Reply:
I must confess that I was wrong there. GridTools needs 16, not 20.

Reply:
Where should I put it then?

Reply:
I just set the vertical domain size to 16 (for all code generation tests) and added a comment to make sure nobody lowers it again. With https://github.com/GridTools/gridtools/pull/1598 merged, there should also be an error thrown by GridTools.